modoboa/lib/sysutils.py

```diff
@@ -22,16 +22,16 @@ def exec_cmd(cmd, sudo_user=None, pinput=None, capture_output=True, **kwargs):
     Run a command using the current user. Set :keyword:`sudo_user` if
     you need different privileges.
 
-    :param str cmd: the command to execute
+    :param collections.abc.Iterable cmd: the command to execute
     :param str sudo_user: a valid system username
     :param str pinput: data to send to process's stdin
     :param bool capture_output: capture process output or not
     :rtype: tuple
     :return: return code, command output
     """
     if sudo_user is not None:
-        cmd = "sudo -u %s %s" % (sudo_user, cmd)
-    kwargs["shell"] = True
+        cmd = ["sudo", "-u", sudo_user] + list(cmd)
+    kwargs["shell"] = False
```

Comment:
I don't think it's a good idea to change this value since you did not modify commands to include absolute pathes. If you disable shell mode, PATH resolution won't be available, right?